### ISSUE TITLE
Meeting does not load when defaultWelcomeMessage and defaultWelcomeMessageFooter are empty

### ISSUE THREAD

**Issue by mittler-works** [bigbluebutton/bigbluebutton#14118]
**Describe the bug**
When you start a Meeting, you will see the three points in the middle for a moment, then the screen is just blue. In the inspector you can see an error `Minified React error #185; visit https://reactjs.org/docs/error-decoder.html?invariant=185 for the full message`.

The React Website says:
```
Maximum update depth exceeded. This can happen when a component repeatedly calls setState inside componentWillUpdate or componentDidUpdate. React limits the number of nested updates to prevent infinite loops.
```

**To Reproduce**
1. Go to your `/etc/bigbluebutton/bbb-web.properties`
2. Make defaultWelcomeMessage and defaultWelcomeMessageFooter empty:
```
...
defaultWelcomeMessage=
defaultWelcomeMessageFooter=
...
```
3. Restart BBB: `bbb-conf --restart`
4. Start a Meeting, you see the error

**Expected behavior**
As in prior versions, there should just be no Welcome Message.

**Actual behavior**
See Bug description.

**Screenshots**
(imagine a dark blue screen)

**BBB version:**
BigBlueButton Server 2.4.1 (2879)

**Desktop (please complete the following information):**
 - OS: Debian 11
 - Browser Chrome V97 and Firefox V91

**Comment by beatwiz** [comment]
Omg this was driving me nuts for the last 6 hours...

**Comment by mittler-works** [comment]
Haha me too, I already thought I was going crazy over this. Glad I could help!

**Comment by hostbbb** [comment]
i always use =<br> to remove the defaults.  not sure when this became an issue.

**Comment by antobinary** [comment]
Thanks for reporting! We're taking a deeper look

**Comment by antobinary** [comment]
This fix will ship with BBB 2.4.2 shortly. Thanks again for reporting! The empty welcome message was a key info!
